common/helpers/colors.js

```diff
@@ -53,21 +56,21 @@ function campCollaborationColor(campCollaboration) {
     return idToColor('', true)
   }
 
-  const loading =
-    campCollaboration._meta?.loading ||
-    (typeof campCollaboration.user === 'function' &&
-      campCollaboration.user()._meta?.loading)
+  const inactive =
```

Comment:
now i would very much like tests for this method.
can also be done later, but i tried to keep all the cases in mind, and i could not.

Reply:
I've added them 

https://github.com/ecamp/ecamp3/pull/5411/files/100af40313b592ced9362a4768f7eea528b4c130#diff-dbb47af7cf605263a60bcfb6b512abafbc27ae482f0925eff56c99f2538d7b0fR45-R57